compiler/src/dmd/common/outbuffer.d

```diff
@@ -73,7 +73,7 @@ struct OutBuffer
     Params:
     filename = zero-terminated name of file to map into memory
     */
-    @trusted this(const(char)* filename)
+    this(const(char)* filename) @trusted
```

Comment:
do we not have a `mallocNew!(T)(Args...)` that we could have be `@trusted` instead?